api/nakadi-event-bus-api.yaml

```diff
@@ -1186,15 +1186,17 @@ definitions:
         description: |
           Offset of the event being pointed to. Regarding its format and guarantees:
 
-          - offsets have a fixed size of 26 characters ranging from 0 to F (Hexadecimals).
+          - offsets have a fixed size of 27 characters ranging from 0 to F (Hexadecimals).
```

Comment:
maybe a regex would be an easier as a documentation? 

`^([0-F]{9})/([0-F]{17})$`